recipes/solaris/LICENSE.txt

```diff
@@ -0,0 +1,201 @@
+Apache License
```

Comment:
It is fine to include the license here. However, you should make an upstream issue/commit to make sure the license is included in the pypi sdist.

Reply:
Thanks - it is.

Reply:
Cool. If the license file is upstream then there is no reason to ship it here and we can delete it.